netclient/config/config.go

```diff
@@ -25,7 +25,6 @@ type ClientConfig struct {
 	Daemon          string         `yaml:"daemon"`
 	OperatingSystem string         `yaml:"operatingsystem"`
 	DebugOn         bool           `yaml:"debugon"`
```

Comment:
This var can be deleted and rework where it is used to log by verbosity to achieve the same functionality